src/components/no-choice-list-item.css

```diff
@@ -0,0 +1,7 @@
+@import '../styles/colors-v2.css';
```

Comment:
Why is this CSS file not with its component file? Looks like this might indicate that the no choice list item shouldn't be in `__internal__` or maybe that we need to allow for CSS files to be in `__internal__`?